### PR TITLE
feat(transformer): error when running tests with no transformer installation

### PR DESCRIPTION
**Pull request by Pmyl**
With this PR when using the placeholder methods (createMock, createMockList, registerMock) without using the transformer it will throw an error that link to this github and the installation page.

I'm bad at writing error messages, please someone help me to write a message that is informative, right now the message is
```
ts-auto-mock is not been configured as a transformer. Please follow the instructions in https://typescript-tdd.github.io/ts-auto-mock/installation
If help is required you can find us on https://typescript-tdd.github.io
```

### CODE REVIEW

**Comment by uittorio** [comment]
I would write something like

``` hey, it looks like ts-auto-mock is not configured correctly! You can find the instructions here https://typescript-tdd.github.io/ts-auto-mock/installation. If you need further assistance feel free to drop a message on slack. (link at the bottom of https://typescript-tdd.github.io/ts-auto-mock)```

**Comment by Pmyl** [comment]
> I would write something like
> 
> 
> 
> ``` hey, it looks like ts-auto-mock is not configured correctly! You can find the instructions here https://typescript-tdd.github.io/ts-auto-mock/installation. If you need further assistance feel free to drop a message on slack. (link at the bottom of https://typescript-tdd.github.io/ts-auto-mock)```

I’ll change it, thanks!

File: src/create-mock-list.ts [hunk 1]
```diff
@@ -1,2 +1,9 @@
 import { DeepPartial } from './partial/deepPartial';
-export declare function createMockList<T extends object>(quantity: number, iterator?: (index: number) => DeepPartial<T>): T[];
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export function createMockList<T extends object>(quantity: number, iterator?: (index: number) => DeepPartial<T>): T[] {
+  throw new Error(`
```
Comment (uittorio) [inline src/create-mock-list.ts:5]: Can we try to extract the Error in one file? something like ts-auto-mock-error.ts.

Comment (Pmyl) [reply src/create-mock-list.ts]: done, also updated error wording

**Pull request merged**